content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,103 @@ description: Upgrading an ambient mode installation with Helm.
+Node cordoning and blue/green node pools are recommended to limit the blast radius of resets on application traffic during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
+## Understanding Ambient Upgrades
+
+All Istio Upgrades involve upgrading the control plane, data plane, and Istio CRDs. Because the Ambient Data Plane is separated into [two components](/docs/ambient/architecture/data-plane), the ztunnel and waypoints, Ambient Upgrades involve separate steps for these components. Upgrading the control plane and CRDs will be covered here in brief, but is essentially identical to the process for these components in sidecar mode, which is covered extensively [here](docs/setup/upgrade/canary/).
+
+Like sidecar mode, ambient mode upgrades can make use of Tags and Revisions to allow fine-grained control over ({{< gloss >}}gateway{{</ gloss >}}) upgrades, including ({{< gloss >}}waypoint{{</ gloss >}})s, with simple controls for rolling back at any point. However, unlike sidecar mode, the Ambient ztunnel runs as a DaemonSet, or per-node proxy, meaning that ztunnel upgrades affect, at minimum, one entire node at a time. While this disruptions should be trivial for most use cases, applications with long-lived TCP connections may be disrupted.  In these cases, we recommend using Node cordoning and draining before upgrading the ztunnel component for a given node. For the sake of simplicity, this document will demonstrate in-place upgrades of the ztunnel, which are less safe, and may involve some downtime.
+
+With that in mind, this guide will expand on the following steps to upgrade Istio in ambient mode:
+
+1. Prerequisites
+1. Upgrade the CRDs
+1. Install the New Control Plane
+1. Upgrade the ztunnel DaemonSet
+1. Upgrade the CNI DaemonSet
+1. Upgrade waypoints and gateways using tags
+1. Uninstall the old Control Plane
+
 ## Prerequisites
 
-1. Update the Helm repository:
+1. Organize your Tags and Revisions
 
-    {{< text syntax=bash snip_id=update_helm >}}
-    $ helm repo update istio
-    {{< /text >}}
+    In order to safely upgrade Ambient, your Gateways and Namespaces should be using the `istio.io/rev` label to specify an Istio Tag which will control the version of the Istio Proxy that is running. We recommend dividing your production cluster into multiple tags to organize your upgrade. All members of a given tag will be upgraded simultaneously, so it is wise to begin your upgrade with your lowest risk applications. We do not recommend referencing revisions directly via labels for upgrades, as this process can easily result in the accidental upgrade of a large number of proxies, and is difficult to segment. To see what tags and revisions you are using in your cluster, see the section on upgrading tags.
```

Comment:
```suggestion
    In order to safely upgrade ambient, your Gateways and Namespaces should be using the `istio.io/rev` label to specify an Istio Tag which will control the version of the Istio Proxy that is running. We recommend dividing your production cluster into multiple tags to organize your upgrade. All members of a given tag will be upgraded simultaneously, so it is wise to begin your upgrade with your lowest risk applications. We do not recommend referencing revisions directly via labels for upgrades, as this process can easily result in the accidental upgrade of a large number of proxies, and is difficult to segment. To see what tags and revisions you are using in your cluster, see the section on upgrading tags.
```

Reply:
Agreed, gateways and namespaces here should be lowercase.

Reply:
And "data plane/control plane" throughout.

I can fix uses of "a mesh with support for ambient mode" etc.
This is just weird while it's gated and not on by default.